build/linux/dist/appdata.xml

```diff
@@ -15,10 +19,13 @@
       to develop and upload code to compatible microcontrollers.
     </p>
   </description>
+
   <screenshots>
     <screenshot type="default" width="624" height="351">http://mavit.fedorapeople.org/appdata/arduino-screenshot.png</screenshot>
     <screenshot width="704" height="396">http://mavit.fedorapeople.org/appdata/arduino-photo.jpg</screenshot>
   </screenshots>
+
   <url type="homepage">http://www.arduino.cc/</url>
-  <updatecontact>arduino.appdata.xml@mavit.org.uk</updatecontact>
-</application>
+
+  <update_contact>arduino.appdata.xml@mavit.org.uk</update_contact>
```

Comment:
I wonder if we should update this to an arduino.cc address (developers mailing list perhaps)? @cmaglie?